packages/provider/provider.ts

```
@@ -0,0 +1,122 @@
+import crypto from "node:crypto";
+
+// 클래스, 심볼, 문자열 모두 토큰으로 사용 가능
+export type InjectionToken<T = any> = Class<T> | symbol | string;
+export type Class<T = any> = {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)